pkg/issuer/acme/dns/dns.go

```diff
@@ -179,6 +180,22 @@ func (s *Solver) solverFor(crt *v1alpha1.Certificate, domain string) (solver, er
 			providerConfig.AzureDNS.ResourceGroupName,
 			providerConfig.AzureDNS.HostedZoneName,
 		)
+	case providerConfig.Inwx != nil:
+		password, err := s.secretLister.Secrets(s.resourceNamespace).Get(providerConfig.Inwx.Password.Name)
+		if err != nil {
+			return nil, fmt.Errorf("error getting inwx password: %s", err.Error())
+		}
+		passwordBytes, ok := password.Data[providerConfig.Inwx.Password.Key]
+		if !ok {
+			return nil, fmt.Errorf("error getting inwx password: password '%s' not found in secret", providerConfig.Inwx.Password.Key)
+		}
+		impl, err = inwx.NewDNSProviderCredentials(
+			providerConfig.Inwx.Username,
+			string(passwordBytes),
```

Comment:
Is it possible for a password to contain a new line? If not, it'd be ideal to trim trailing newlines from the secret resource (similar to #391)